app/controllers/public/sessions.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+import { orderBy, groupBy } from 'lodash';
+
+const { Controller, computed } = Ember;
+
+export default Controller.extend({
+  sessions: [{ shortAbstract: 'An introduction to the event', id: '1', title: 'Welcome to FOSSASIA', startAt: '10:35 AM / 17-03-2017', track: { color: 'green', fontColor: 'green', id: 0, name: 'Track 1' }, microlocation: { id: 0, name: 'Room 1' }, speakers: [{ shortBiography: 'Works for ORG 1', id: 0, city: 'Delhi', name: 'Arnold Singh', speakingExperience: 'GSOC 2015', organisation: 'ORG 1', longBiography: '', photo: { 'iconImageUrl': '' } }] },
```

Comment:
The model's output is gonna be a js date object. (It will parse the API Response and give a js output) ... And we shouldn't do the string conversion in the model ... because the string date format we require might vary depending on where the session is being displayed in the UI.